app/views/pages/deposit.html.erb

```diff
@@ -0,0 +1,5 @@
+<div class="deposit_page page container">
```

Comment:
What are some of these classes for like `deposit_page`? There is nothing in the stylesheets and its not being reference elsewhere? Just extra markup that's not needed, therefore should just remove this